docs/devGuide/development/migratingToTypeScript.md

```diff
@@ -60,6 +60,11 @@ We have decided on structuring the commits as two commits: a "Rename" commit and
 4. **Commit the staged changes** with the message `Rename <file-or-folder> to TypeScript`.
    <br>
    For example: `Rename core/src/html to TypeScript`.
+   
+   <box type="info">
+   Any git hooks that have been installed will automatically build the backend as part of the hook. If you encounter any compilation errors while committing and/or pushing the "Rename" commit due to git hooks, skip the hooks using the `--no-verify` flag.
```

Comment:
To be precise, it's not "any" git hooks, it's just the git hooks that we have written for MarkBind, which runs pre-commit. Let's make this clearer, and link to the docs that explain the git hook that we have.

Reply:
Updated in [dba39d1](https://github.com/MarkBind/markbind/pull/2151/commits/dba39d1ce84ce271a31f6c753110d6e966e92108) - changes can be seen [here](https://deploy-preview-2151--markbind-master.netlify.app/devguide/development/migratingtotypescript#step-2-and-quot-rename-and-quot-commit)!